torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -262,6 +299,21 @@ std::string shapesToStr(const std::vector<std::vector<int64_t>>& shapes) {
   return oss.str();
 }
 
+std::string dtypesToStr(const std::vector<std::string>& types) {
+  std::ostringstream oss;
+  oss << "[";
```

Comment:
I think you can use fmt::format() to simplify this?

Something like:

return fmt::format("["\{}\"]", types fmt::join(types, "\", \"));

Reply:
from the time being, the profiler_kineto doesn't include fmt dependency. Is it ok to introduce additional dependencies for all platforms?

Reply:
yeah, try adding `#include <fmt/format.h>` i think it should work out of the box - pytorch already depends on fmt

Reply:
just find another way to simply the logic by using ostringstream without introduce the fmt dependency

Reply:
it seems fmt::join fail in [pytorch_linux_xenial_py3_clang5_asan_test2]( https://circleci.com/api/v1.1/project/github/pytorch/pytorch/12606907/output/107/0?file=true&allocation-id=607e76b550c532340a9ce1c6-0-build%2F1E72253E)